locale/en_US/manager.po

```diff
@@ -721,6 +721,33 @@ msgstr "Notification of Author Submission"
 msgid "manager.setup.notifications.copySpecifiedAddress"
 msgstr "Send a copy to this email address"
 
+msgid "manager.setup.submissionsNotifications"
+msgstr "Notifications"
+
+msgid "manager.setup.notifications.copySubmissionAckPrimaryContact"
+msgstr "Notify Primary Contact"
+
+msgid "manager.setup.notifications.copySubmissionAckPrimaryContact.description"
+msgstr "Send a copy of the submission acknowledgement email to this journal's primary contact."
+
+msgid "manager.setup.notifications.copySubmissionAckPrimaryContact.disabled.description"
+msgstr "No primary contact has been defined for this journal. You can enter a primary contact in the <a href="{$url}">journal settings</a>"
```

Comment:
This, and the locale key above it, need to go into OJS because they use the word "journal".

Also, add a `.` at the end of this, so `settings</a>."`